### DISCUSSION TITLE
Control HA Devices from HC3

### DISCUSSION THREAD

**Discussion by sergiobaiao** [alexander-vitishchenko/hc3-to-mqtt#51] [Ideas]
Hi there,

I was talking with you about the possibility, but found out that it already exists.

Here is a link for future reference:

[https://forum.fibaro.com/topic/53397-quickapp-homeassistant/](https://forum.fibaro.com/topic/53397-quickapp-homeassistant/)

**Comment by alexander-vitishchenko** [comment]
Oh that's good! Thanks for sharing the link.
